tests/requirements/base.txt

```diff
@@ -2,3 +2,6 @@
 django-app-helper
 tox
 coverage
+black
```

Comment:
We're using ruff for linting and formatting, so we could merge this with black and the old linting setup and tackle ruff as a separate task. But just wanted to raise this here.

It's in pre-commit for django-cms itself; https://github.com/django-cms/django-cms/blob/develop-4/.pre-commit-config.yml#L41

Reply:
I can also throw out the commits for "including black"/"formatting running black" if and maybe switch it with `ruff` for the development process?
Or what would be the workflow you'd want?

Reply:
I don't mind the code being formatted with black because the ruff formatter is about 97% the same.

The workflow/process around this area on the whole is using pre-commit hooks;

https://github.com/django-cms/django-cms/blob/develop-4/.pre-commit-config.yml

Combined with a github action that runs the linter;
https://github.com/django-cms/django-cms/blob/develop-4/.github/workflows/linters.yml

Reply:
Alright, then I'd leave that 'as is' for now.